pkg/internal/checkup/executor/console/console.go

```diff
@@ -105,18 +105,15 @@ func RetValue(retcode string) string {
 // waiting for the batch to return with a response until timeout.
 // It validates that the commands arrive to the console.
 // NOTE: This functions inherits limitations from `expectBatchWithValidatedSend`, refer to it for more information.
-func SafeExpectBatchWithResponse(serialConsoleClient vmiSerialConsoleClient,
-	vmiNamespace,
-	vmiName string,
-	expected []expect.Batcher,
+func (e Expecter) SafeExpectBatchWithResponse(expected []expect.Batcher,
 	timeout time.Duration) ([]expect.BatchRes, error) {
-	expecter := NewExpecter(serialConsoleClient, vmiNamespace, vmiName)
-	gGExpect, err := expecter.spawnConsole(timeout)
+	expecter, err := e.spawnConsole(timeout)
```

Comment:
AFAIU, this `expecter` is of the lower level type, and not the custom type that was defined in a previous commit.
Please consider renaming it.

Reply:
yeah I agree. I'll keep the prior wording gGExpect